index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+    <div class="container">
+        <nav>
+            <div id="logo">
+                <h1>My Logo</h1>
+            </div>
+            <div>
+                <i class="fa fa-bars fa-lg"></i>
+            </div>
+        </nav>
+        <header>
+            <div class="header-container">
+                <div>
+                    <h2>I'm Mariane</h2>
```

Comment:
- [ ] Kindly change this `h2` tag to the `h1` tag. This is the main title of the webpage. If you want google (and other search engines) to know about a specific topic, in this case, you should use an `h1` tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more information. 💻